app/controllers/catalog_controller.rb

```diff
@@ -114,8 +114,9 @@ def self.chf_search_fields
     # solr fields to be displayed in the index (search results) view
     #   The ordering of the field names is the order of the display
     config.add_index_field solr_name("title", :stored_searchable), label: "Title", itemprop: 'name', if: false
+    config.add_index_field solr_name("additional_title", :stored_searchable), label: "Additional Title", itemprop: 'name', if: :present?
```

Comment:
> Ha, okay, so I have put all kinds of junk in itemprop inappropriately. I'll open an issue to revise those.

Seriously, I don't think it matters, I don't know it's a good use of time to deal with that now. I'd really like to entirely redo this page, when we have a chance, and deal with it all then. But do you what you feel is best of course!